railway/railway.md

```diff
@@ -0,0 +1,81 @@
+
+Create your first user by going to `https://app-name.up.railway.app` or your custom domain.
+
+![Welcome to Coder](https://raw.githubusercontent.com/coder/blogs/coder-on-railway/posts/coder-on-railway/static/coder_setup.png)
+
+### 3. Create your first template
+
+[**Templates**](https://coder.com/docs/v2/latest/templates): Templates are written in Terraform and describe the infrastructure for workspaces. Coder provides a set of starter templates to help you get started.
+
+Choose a template to set up your first workspace. You can also [create your own templates](https://coder.com/docs/v2/latest/templates) to define your custom infrastructure with your preferred cloud provider.
+![starter templates](https://raw.githubusercontent.com/coder/blogs/coder-on-railway/posts/coder-on-railway/static/starter_templates_welcome.png)
+
+### 4. Create your first workspace
+
+[**Workspaces**](https://coder.com/docs/v2/latest/workspaces): Workspaces contain the IDEs, dependencies, and configuration information needed for software development. You can create workspaces from templates. Here wea are showing the workspaces created from the Fly.io starter template in action.
+![fly.io workspace](https://raw.githubusercontent.com/coder/blogs/coder-on-railway/posts/coder-on-railway/static/fly_workspace.png)
+
+## Documentation
+
+Browse our [**docs**](https://coder.com/docs/v2) or visit a specific section below:
+
+[**Templates**](https://coder.com/docs/v2/latest/templates) | [**Workspaces**](https://coder.com/docs/v2/latest/workspaces) | [**IDEs**](https://coder.com/docs/v2/latest/ides) | [**Administration**](https://coder.com/docs/v2/latest/admin) | [**Enterprise**](https://coder.com/docs/v2/latest/enterprise)
+
+## Community and Support
+
+Feel free to [open an issue](https://github.com/coder/coder/issues/new) if you have questions, run into bugs, or have a feature request.
+
+[Join our Discord](https://discord.gg/coder) to provide feedback on in-progress features, and chat with the community using Coder! [![discord](https://img.shields.io/discord/747933592273027093?label=discord)](https://discord.gg/coder)
+
+## Related
+
+We are always working on new integrations. Feel free to open an issue to request an integration. Contributions are welcome in any official or community repositories.
+
+### Official
+
+- [**VS Code Extension**](https://marketplace.visualstudio.com/items?itemName=coder.coder-remote): Open any Coder workspace in VS Code with a single click
+- [**JetBrains Gateway Extension**](https://plugins.jetbrains.com/plugin/19620-coder): Open any Coder workspace in JetBrains Gateway with a single click
+
+### Community
+
+- [**Coder GitHub Action**](https://github.com/marketplace/actions/update-coder-template): A GitHub Action that updates Coder templates
+- [**Various Templates**](https://github.com/coder/coder/examples/templates/community-templates.md): Hetzner Cloud, Docker in Docker, and other templates the community has built.
```

Comment:
I'm 50/50 on whether we should keep this section. Can you make this call @matifali?

We could also remove all these headers (Documentation, community, etc) and just have bullet points

- [Coder on GitHub](#)
- [Coder Discord](#)
- [Coder docs](#)

Reply:
I think can remove the Related section and keep the Official and Community.
Your suggestion is also good to just clean it up and give links.